### PR TITLE
feat(integrations): Add Support for Elevenlabs

### PR DESCRIPTION
**Pull request by Shubhamai**
## Describe your changes
- Added [Elevenlabs](http://elevenlabs.io) support.

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/integrations/all/elevenlabs.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Elevenlabs
+sidebarTitle: Elevenlabs
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/elevenlabs.mdx]: The appropriate naming seems to be `ElevenLabs` if you can update all ref 👍🏻

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -1424,6 +1424,22 @@ egnyte:
             description: The subdomain of your Egnyte account
             pattern: '^[a-z0-9_-]+$'
 
+elevenlabs:
+    display_name: Eleven Labs
+    categories:
+        - dev-tools
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:1519]: Needs to be added to dev-tools.mdx

File: packages/shared/providers.yaml [hunk 3]
```diff
@@ -1424,6 +1424,22 @@ egnyte:
...
+    auth_mode: API_KEY
+    proxy:
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:1521]: Can you add a `verification` prop. It should a simple call to an endpoint accessible to anyone with this api key

**Review by bodinsamuel — COMMENTED** [review]
Thanks for your contribution 🙏🏻 
Just a few comments and it will be good

File: docs-v2/integrations/all/elevenlabs.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+API configuration: [`elevenlabs`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (API Key)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | 🚫 (time to contribute: &lt;48h) |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/elevenlabs.mdx]: Everything should be green here

**Comment by Shubhamai** [comment]
Hi, thanks for the review, just completed all the review and should be good to go!

**Comment by hassan254-prog** [comment]
Small addition to @bodinsamuel , please add the following [logo](https://codefile.io/f/KgM00ZqtxP)

**Review by hassan254-prog — APPROVED** [review]

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**